# Water Conflicts in the Colorado River Basin


Data is collected from 9 recorded public meetings of water control boards and other offical organizations.  Data was also collected from newspaper articles from areas in the Colorado River Basin.

 Data collected ranges from January 1, 2005 to December 31, 2021.

The authors perceive the dataset to be useful for identifying and giving context for conflict and other water-related crises regarding water resource management in the Colorado River Basin.  This contect may be especially useful for USGS water availability reports.

The data was accessed on October 6, 2025 from Holloman et al. (2023).

Holloman, D.V., Hines, M.K., and Zoanni, D.K., 2023, Coded Water Conflict and Crisis Events in the Colorado River Basin, Derived from LexisNexis search 2005-2021: U.S. Geological Survey data release, (https://doi.org/10.5066/P9X6WR7J).

In [4]:
import pandas as pd
import numpy as np

df = pd.read_csv('../data/Colorado River Basin Water Conflict Table.csv')

## Preliminary Data Exploration

In [3]:
#Set pandas to display all columns in the data frame
pd.set_option('display.max_rows', None)

In [ ]:
# Identify the types of data in each column and 
# how many observations have data for each
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 268 entries, 0 to 267
Data columns (total 48 columns):
 #   Column                                        Non-Null Count  Dtype  
---  ------                                        --------------  -----  
 0   Event                                         268 non-null    int64  
 1   Search Source                                 268 non-null    object 
 2   Newspaper                                     268 non-null    object 
 3   Article Title                                 268 non-null    object 
 4   Duplicate                                     267 non-null    object 
 5   Report Date                                   267 non-null    object 
 6   Report Year                                   265 non-null    float64
 7   Event Date                                    248 non-null    object 
 8   Event Day                                     18 non-null     float64
 9   Event Month                                   212 non-null    flo

In [ ]:
# Identify the number of observations and number of columns in the data frame
df.shape

(268, 48)

There are a total of 268 observations in the dataset

In [ ]:
#Identify the names of the columns in the dataframe
df.columns

Index(['Event', 'Search Source', 'Newspaper', 'Article Title', 'Duplicate',
       'Report Date', 'Report Year', 'Event Date', 'Event Day', 'Event Month',
       'Event Year', 'Conflict Present', 'Crisis Present', 'Basin', 'HUC6',
       'HUC2', 'Place', 'County', 'County FIPS', 'State', 'State FIPS',
       'Urban or Rural', 'Issue Type', 'Event Summary', 'Stakeholders',
       'Intensity Value', 'Comments', 'Related Observation Themes',
       'Article Text Search - water quality',
       'Article Text Search - invasive species',
       'Article Text Search - conservation', 'Article Text Search - drought',
       'Article Text Search - flood',
       'Article Text Search - ground water depletion',
       'Article Text Search - depletion',
       'Article Text Search - infrastructure',
       'Article Text Search - fish passage',
       'Article Text Search - instream water rights',
       'Article Text Search - water rights',
       'Article Text Search - intergovernmental',
       '

In [ ]:
#Identify the number of unique values are in each column
df.nunique()

Event                                           268
Search Source                                     6
Newspaper                                        45
Article Title                                   267
Duplicate                                         2
Report Date                                     235
Report Year                                      18
Event Date                                       52
Event Day                                        12
Event Month                                      12
Event Year                                       21
Conflict Present                                  2
Crisis Present                                    2
Basin                                            25
HUC6                                             22
HUC2                                              7
Place                                           135
County                                            3
County FIPS                                       3
State       